modules/virt-example-nmstate-IP-management.adoc

```diff
@@ -100,7 +100,7 @@ Setting a DNS configuration is comparable to modifying the `/etc/resolv.conf` fi
 
 [IMPORTANT]
 ====
-You cannot use `br-ex` bridge, an OVNKubernetes-managed Open vSwitch bridge, as the interface when configuring DNS resolvers.
+A bare-metal platform with an NMState configuration that sets a `br-ex` bridge, an OVNKubernetes-managed Open vSwitch bridge, as the interface only supports configuring DNS resolvers. If you also used `configure-ovs.sh` to set a `br-ex` bridge for your network, this can cause conflicting issues with the `br-ex` bridge that you set by using the NMState configuration method.
```

Comment:
This is a bit confusing and I'm not sure it's correct as written. I think I would keep most of the original message and add the new exception:

"You cannot use `br-ex` bridge, an OVNKubernetes-managed Open vSwitch bridge, as the interface when configuring DNS resolvers, unless you configured br-ex manually using [link to day 1 process for NMState br-ex configuration]."

For now the emphasis should still be on prohibiting the use of br-ex since the default configuration has that limitation. Users who created br-ex manually are hopefully aware that they have more flexibility so it's not as important to make them aware of that. We just need to mention it so we aren't contradicting ourselves here.

Reply:
:heavy_check_mark: